compiler.ts

```diff
@@ -212,6 +218,15 @@ function initGlobalFuns(funs: Array<string>, env: GlobalEnv): Array<string> {
   return inits;
 }
 
+function myMemForward(n: number): Array<string> {
+  const forward: Array<string> = [];
+  forward.push(`;; update the heap ptr`);
+  forward.push(`(i32.const 0)`);
+  forward.push(`(i32.add (i32.load (i32.const 0)) (i32.const ${n * 4}))`);
+  forward.push(`(i32.store)`);
+  return forward;
+}
+
```

Comment:
What is the purpose of `myMemForward()`?

Is it for runtime allocation or static allocation?